tests/core/test_stddef.cpp

```diff
@@ -25,9 +25,8 @@ void abort(void);
 #endif
 
 int main() {
-  // max_align_t on wasm backend is 16 due to sizeof(long double) being 16.
-  // TODO: change this to just 8 after the new ABI rolls in.
-  if (_Alignof(max_align_t) != 16 && _Alignof(max_align_t) != 8)
+  // max_align_t on wasm backend is 8.
+  if (_Alignof(max_align_t) != 8)
     abort();
```

Comment:
While you are there maybe we should replace this with `assert(_Alignof(max_align_t) == 8);`.   It makes that failures easier to read when we use assert.